Packs/OTRS/README.md

```diff
@@ -1 +1,3 @@
+Note: Support for this Pack was moved to Partner starting Oktober 05, 2023. In case of any issues arise, please contact the Partner directly at sales@otrs.com

```

Comment:
The pack readme is a better place to provide general information about your pack and product what it does and what are its use cases. 
The place to specify the partner support is in the pack_metadata.json that you did.

```suggestion
Support for this Pack is provided by OTRS. For additional info on the product please visit: https://otrs.com/


```